Dockerfile

```
@@ -7,9 +7,19 @@ RUN echo "deb http://packages.cloud.google.com/apt cloud-sdk-jessie main" | tee
       echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
       curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
       curl -sL https://deb.nodesource.com/setup_11.x | bash - && \
+      wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+          && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
```

Let's lose the indentation as well, like above, and leave `&&` at the end of the line as elsewhere
```suggestion
    sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' && \
```